README.md

```diff
@@ -323,6 +323,33 @@ const personFactory = Factory.defineUnregistered<Person>(() => ({
 const person = personFactory.build();
 ```
 
+### Rewind Sequence
+
+A factory's sequence can be rewound with `rewindSequence()`.
+This sets the sequence back to its original starting value.
+
+Given the following factory
+
+```typescript
+export default Factory.define<User>(({ sequence }) => ({
+  email: `person${sequence}@example.com`,
+}));
+```
+
+You can rewind a factory's sequence at your discretion
+
+```typescript
+import { factories } from './factories';
+
+factories.user.build(); // { email: 'person0@example.com' }
+factories.user.build(); // { email: 'person1@example.com' }
+factories.user.build(); // { email: 'person2@example.com' }
```

Comment:
Do the second and third `build()` calls add value? I'd recommend condensing this to:

```typescript
factories.user.build(); // { email: 'person0@example.com' }
factories.user.rewindSequence();
factories.user.build(); // { email: 'person0@example.com' }
```

Reply:
I've updated this to build two objects before the`rewindSequence` is called. I think this helps illustrate the effect of `rewindSequence` more clearly as it shows the sequence value is set back to the starting value after it's last value of `2`